gensim/scripts/segment_wiki.py

```diff
@@ -61,10 +61,7 @@
 import multiprocessing
 import re
 import sys
-try:
-    from xml.etree import cElementTree as ET
-except ImportError:
-    from xml.etree import ElementTree as ET
+from xml.etree import ElementTree as ET

```

Comment:
Not related to your PR, but I'm not a fan of such "shortcuts".

Can you please change this to use the full `ElementTree` or `xml.etree. ElementTree`?

Reply:
Do you mean like this?

```python
import xml.etree.ElementTree as ET
...
elem = ET.fromstring(page_xml)
```

Or like this?

```python
import xml.etree.ElementTree
...
elem = ElementTree.fromstring(page_xml)
```

---

I'll note the former is used in the docs and quite common:

```python
import xml.etree.ElementTree as ET
tree = ET.parse('country_data.xml')
root = tree.getroot()
```

But happy to change to whichever you wish.

Reply:
`from xml.etree import ElementTree`. I don't want our code sprinkled with 2-letter acronym puzzles like `ET`.

They may be an obvious shorthand to the module's author, but not anybody else reading the code.

Reply:
Updated!